app/assets/stylesheets/application.scss

```diff
@@ -171,3 +169,13 @@ li {
   position: fixed;
   left: 50px;
 }
+
+table {
+  position: relative;
+  margin: auto;
+}
+
+#system_chart {
```

Comment:
Avoid using id selectors<br>Selector `system_chart` should be written in lowercase with hyphens